single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -899,6 +902,47 @@ private void drawFeederInfo(String prefixId, String feederNodeId, List<Point> po
         root.appendChild(g);
     }
 
+    protected void insertBusInfo(String prefixId,
+                                 Point point,
```

Comment:
`point` is the shift due to the voltage level coordinates, so can you choose a clearer name? like vlPoint or shift or vlShift?